src/System.Private.Xml/src/System/Xml/Resolvers/XmlPreloadedResolver.cs

```diff
@@ -351,7 +351,7 @@ public void Add(Uri uri, Stream value)
                 }
                 int size = checked((int)ms.Position);
                 byte[] bytes = new byte[size];
-                Array.Copy(ms.ToArray(), 0, bytes, 0, size);
+                Array.Copy(ms.ToArray(), bytes, size);
```

Comment:
Since we're already touching this code should we delete these lines 352 - 354, replacing them instead with `byte[] bytes = ms.ToArray();`? Not sure if something like that would be in scope of this PR.

Reply:
This code does seem wonky, but I have no idea what the test coverage of this looks like, so I'm going to leave it as-is.  My changes in this PR were entirely mechanical.  Thanks, though.